RELEASENOTES.md

```diff
@@ -18,7 +18,40 @@ contributors - thank you!
 Questions about a release or the release notes? Ask on the
 [Archivematica User Group](https://groups.google.com/g/archivematica)!
 
-## (Current Release) Archivematica 1.17.0 and Storage Service 0.23.0
+## (Current Release) Archivematica 1.17.1
+
+*Release date*: TBD
+
+[Original release
+notes](https://wiki.archivematica.org/Archivematica_1.17.1_release_notes).
```

Comment:
At some point, we could adopt reference-style links in this document. Beyond cosmetics, the main benefit is improved readability when viewing the document as plain text.

```suggestion
[Original release notes].

[Original release notes]: https://wiki.archivematica.org/Archivematica_1.17.1_release_notes
```

Reply:
I've used reference-style links in the new section https://github.com/artefactual/archivematica/pull/2058/commits/25fb51230768912d76214074f7710406e33835d1